src/ui/components/AlertDialogs/DeleteItemDialog.tsx

```diff
@@ -47,8 +55,24 @@ export const DeleteItemDialog = memo(function DeleteItemDialog(
             </ProductButton>
             <ProductButton
               backgroundColor={colors.errorPrimary}
-              onClick={() => {
-                /*USUWANIE*/
+              onClick={async () => {
+                try {
+                  const updatedSchema = await fetcher(
+                    API_URL + '/api/schemas/deleteItem',
+                    {
+                      body: {
+                        schemaId: props.schemaId,
+                        itemId: props.itemToDelete,
+                      },
+                    }
+                  )
+                  dispatch(updateSchema(updatedSchema))
+                } catch (error) {
+                  console.log(error)
```

Comment:
chcemy logować czy wywołać error?

Reply:
Jeśli przez wywołanie errora rozumiesz wyjebanie apki przez błąd, to nie xD

Reply:
zostańmy przy logach xd